### DISCUSSION TITLE
Verbrauchsdatenbank

### DISCUSSION THREAD

**Discussion by Tollentino** [cyberthom42/vaillant-arotherm-plus#14] [Ideas]
Also ich habe jetzt ein Project erstellt (öffentlich zugänglich) aber ich kann es in diesem Repository nicht in Projects verlinken.
Ich vermute, die Funktion Projects ist eher dazu da um Tickets (Issues/Tasks) besser zu tracken. Also quasi eine Managementfunktion.

Evtl ist eine Excel doch besser?

**Comment by cyberthom42** [comment]
Ich komme heute nicht dazu, ich schaue mir das morgen an. XLS bzw. CSV ist natürlich immer eine Möglichkeit. Es ist auch möglich die CSV in github als Tabelle darstellen zu lassen, siehe ebusd github

**Reply by Tollentino** [reply]
Hab jetzt mal eine Excel hochgeladen. csv ginge natürlich auch, aber pflegen wird es wohl 80% dann eh in Excel oder?
Gut du dumpst es dann wahrscheinlich direkt aus deinem logger... Aber die Hoffnung ist ja, dass hier noch ein paar mehr Normalos reinkommen...
[Verbrauchsdaten_monatlich.xlsx](https://github.com/cyberthom42/vaillant-arotherm-plus/blob/7f2fcdf0baec882fcf8ae70ac874e4dfbadc0119/Verbrauchsdaten_monatlich.xlsx)

**Comment by cyberthom42** [comment]
Wir könnten die Tabelle als CSV abspeichern, so kann man sie auch direkt anzeigen lassen und mit filtern arbeiten:
[https://github.blog/2013-08-22-see-your-csvs/](url)
